src/main/java/org/fundacion/jala/converter/view/converter/AudioConverterInterface.java

```diff
@@ -80,43 +84,42 @@ public AudioConverterInterface() {
      * @param e event of the JButton.
      */
     @Override
-    public void actionPerformed(final ActionEvent e) {
-        LOGGER.info("Start");
-        JOptionPane.showMessageDialog(this, "File Path: "
-                + file.getOriginFilePath()
-                + "\nConvert to: "
-                + audioSelect.getConvertTo()
-                + "\nQuality: "
-                + quality.getQualityAudio()
-                + "\nVolume: "
-                + settings.getVolume()
-                + "\nAudio Channel: "
-                + settings.getAudioChannel()
-                + "\nHz: "
-                + settings.getHz()
-                + "\nwith metadata: "
-                + settings.isMetadata());
+    public void actionPerformed(final ActionEvent e)  {
         try {
-            LOGGER.info("Execute Try");
+            JOptionPane.showMessageDialog(this, "File Path: "
+                    + file.getOriginFilePath()
+                    + "\nConvert to: "
+                    + audioSelect.getConvertTo()
+                    + "\nQuality: "
+                    + quality.getQualityAudio()
+                    + "\nVolume: "
+                    + settings.getVolume()
+                    + "\nAudio Channel: "
+                    + settings.getAudioChannel()
+                    + "\nHz: "
+                    + settings.getHz()
+                    + "\nwith metadata: "
+                    + settings.isMetadata()
+                    + "\nChecksum: "
+                    + getFileChecksum(file.getOriginFilePath()));
             callRequest();
-        } catch (Exception ex) {
-            LOGGER.error("Execute Exception to metaData conversion");
-            ex.printStackTrace();
+        } catch (IOException ioException) {
+            ioException.printStackTrace();
+        } catch (NoSuchAlgorithmException noSuchAlgorithmException) {
+            noSuchAlgorithmException.printStackTrace();
         }
-        LOGGER.info("Finish");
     }
-
     /**
      * Obtains the request
      * @throws IOException
      */
     private void callRequest() throws IOException {
-        LOGGER.info("start");
```

Comment:
The start of the logger shouldn't be deleted

Reply:
ok